exercises/nth-prime/nth_prime_test.rb

```diff
@@ -22,13 +25,35 @@ def test_sixth_prime
 
   def test_big_prime
     skip
-    assert_equal 104_743, Prime.nth(10_001)
+    assert_equal 104743, Prime.nth(10001)
   end
 
-  def test_weird_case
+  def test_there_is_no_zeroth_prime
     skip
     assert_raises ArgumentError do
       Prime.nth(0)
     end
   end
```

Comment:
The `json` file generates the `false` case. After it generated that I changed it to raise the exception.

I wasn't sure whether to leave it `false` or change it to this.